Source/WebCore/html/canvas/WebGL2RenderingContext.cpp

```diff
@@ -803,6 +803,16 @@ WebGLAny WebGL2RenderingContext::getInternalformatParameter(GCGLenum target, GCG
         return nullptr;
     }
 
+    // These formats are never exposed to WebGL apps but may be accepted by ANGLE.
+    switch (internalformat) {
+    case 0x6AC0: // GL_BGRA8_SRGB_ANGLEX
```

Comment:
Instead of numeric constants in the code, could you add these to the GraphicsContextGL enum like the rest of them?